app/libraries/config.ts

```diff
@@ -1,6 +1,6 @@
 import { Library } from '.'
 
-export const configProject: Library = {
+export const configProject = {
```

Comment:
`satisfies` is generally nicer but I'm aware that it's slower then doing the explicit typing. willing to revert.

Reply:
wait, this is application code not a library. I think it's fine then.